src/app/products/recurring-deposit-products/view-recurring-deposit-product/view-recurring-deposit-product.component.html

```diff
@@ -0,0 +1,415 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" fxLayout.lt-md="column">
+
+        <h2 class="mat-h2" fxFlexFill>{{ recurringDepositProduct.name }}</h2>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <h3 class="mat-h3" fxFlexFill>Details</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill *ngIf="recurringDepositProduct.description">
+          <span fxFlex="40%">Description:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.description }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Short Name:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.shortName }}</span>
+        </div>
+
+        <h3 class="mat-h3" fxFlexFill>Items</h3>
```

Comment:
Please check the preview of http://localhost:4200/products/fixed-deposit-products/create as an example and accordingly update this into the correct sections. Check for any changes to the display like for Deposit Amount and Interest Rate Chart and update that as well. Take reference from View Loan/Saving/Share product as well so we can make the code consistent with the web app.

Reply:
I have updated all the sections making the reference as provided above.